data/random-teams.ts

```diff
@@ -1638,9 +1634,31 @@ export class RandomTeams {
 			}
 		}
 
+		// Consolidate mostly-cosmetic formes, at least for the purposes of Random Battles
+		if (species.baseSpecies === 'Basculin') {
+			forme = 'Basculin' + this.sample(['', '-Blue-Striped']);
+		}
+		if (species.baseSpecies === 'Dudunsparce') {
+			forme = 'Dudunsparce' + this.sample(['', '-Three-Segment']);
+		}
+		if (species.baseSpecies === 'Magearna') {
+			forme = 'Magearna' + this.sample(['', '-Original']);
+		}
+		if (species.baseSpecies === 'Maushold') {
+			forme = 'Maushold' + this.sample(['', '-Four']);
+		}
 		if (species.baseSpecies === 'Pikachu') {
 			forme = 'Pikachu' + this.sample(['', '-Original', '-Hoenn', '-Sinnoh', '-Unova', '-Kalos', '-Alola', '-Partner', '-World']);
 		}
+		if (species.baseSpecies === 'Polteageist') {
+			forme = 'Polteageist' + this.sample(['', '-Antique']);
+		}
+		if (species.baseSpecies === 'Sinistcha') {
+			forme = 'Sinistcha' + this.sample(['', '-Masterpiece']);
+		}
+		if (species.baseSpecies === 'Zarude') {
+			forme = 'Zarude' + this.sample(['', '-Dada']);
+		}
```

Comment:
this could probably get consolidated into a function

Reply:
If we knew the best way to do so, it would have been done. Any ideas?

Reply:
I've moved all the forme selection code into their own function.

Reply:
> The point wasn’t to move all this code into a function, the point was to write a generic function that handles randomizing one Pokémon’s forme among its cosmetic equivalents so you don’t need all this nearly identical code repeated for each species.

oh right. I can do that for the other ones, but not Pikachu or Basculin, since they don't sample from all their alt formes.